sessionctx/stmtctx/stmtctx_test.go

```diff
@@ -107,7 +107,11 @@ func TestWeakConsistencyRead(t *testing.T) {
 		ctx := context.WithValue(context.Background(), "CheckSelectRequestHook", func(req *kv.Request) {
 			require.Equal(t, req.IsolationLevel, isolationLevel)
 		})
-		tk.Session().Execute(ctx, sql)
+		rss, err := tk.Session().Execute(ctx, sql)
```

Comment:
use ```tk.Exec``` ?

Reply:
The hook function is passed by context parameter in `Session.Execute`, but `tk.Exec` uses inner context.